source/FreeRTOS_BitConfig.c

```diff
@@ -237,7 +237,7 @@ void vBitConfig_write_uc( BitConfig_t * pxConfig,
  * @param[in] pxConfig: The structure containing a copy of the bits.
  * @param[in] ucValue: The byte to be written.
  *
- * @return There is no return value. If the operation has failed,
+ * There is no return value. If the operation has failed,
```

Comment:
Is @return not needed anymore?

Reply:
No, as the function `vBitConfig_write_uc` doesnt return anything we dont need that